package.json

```diff
@@ -1,18 +1,47 @@
 {
   "name": "stylelint-config-upleveled",
-  "version": "0.0.1",
-  "description": "",
-  "main": "index.js",
-  "files": [
-    "index.js"
-  ],
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "version": "1.0.0",
+  "description": "UpLeveled Stylelint defaults for styling in CSS, SCSS, Sass, CSS Modules and CSS-in-JS.",
   "keywords": [
     "stylelint",
     "stylelint-config"
   ],
-  "author": "Karl Horky",
-  "license": "ISC"
+  "repository": "upleveled/stylelint-config-upleveled",
+  "license": "MIT",
+  "author": "Karl Horky <karl.horky@gmail.com>",
+  "type": "module",
+  "main": "index.cjs",
+  "files": [
+    "index.cjs",
+    "bin",
+    "templates",
+    "tsconfig.base.json",
+    "stylelint-error-example.webp"
+  ],
+  "stylelint": {
+    "extends": [
+      "."
+    ]
+  },
+  "eslint": {
+    "extends": [
+      "."
+    ]
+  },
```

Comment:
for these new things introduced, maybe you can add a PR comment here describing why we need them vs the old config

would be great to not have extra tool config in the `package.json` file

also do this for other lines in this PR which are a bit unclear / new

Reply:
https://github.com/upleveled/stylelint-config-upleveled/pull/3/commits/de46f3805026aacb331ad4514f1292537904d8c4